src/Kdyby/Doctrine/DI/OrmExtension.php

```diff
@@ -709,8 +703,8 @@ protected function processRepositories()
 				$entityArgument = $boundEntity;
 
 			} else {
+				// TODO Obtain an Entity FQN w/o ClassMetadataFactory::getAllMetadata invoking or should it be done another faster way?
```

Comment:
It's complicated and currently I prefer #274 in my projects.

Maybe I'll try do this later, sorry.